target/linux/ipq40xx/patches-5.4/901-arm-boot-add-dts-files.patch

```diff
@@ -58,6 +58,8 @@ Signed-off-by: John Crispin <john@phrozen.org>
 +	qcom-ipq4019-e2600ac-c2.dtb \
 +	qcom-ipq4019-habanero-dvk.dtb \
 +	qcom-ipq4019-pa2200.dtb \
++	qcom-ipq4019-r619ac-64m.dtb \
```

Comment:
While the 5.4 patches are sticking around, ipq40xx no longer has 5.4 marked as a supported kernel.
Can you please update this to 5.10?

Reply:
Updated.